simple-xml/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for simple-xml 0.0
+// Project: https://github.com/AirAsiaExpedia/node-simple-xml
+// Definitions by: Vitaliy Isikov <https://github.com/notVitaliy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+
+declare namespace simple.xml {
+  namespace XML {
```

Comment:
This will result in a nested namespace, so you'd have to use `import XML = require("simple-xml"); XML.XML.parse(...)`.
Instead, just directly export these functions (`export function stringify(obj: {}): string; export function parse(data: string): {}`).
And please add tests so we can catch this kind of error.